flowdb/sql/07_schema_cache.sql

```diff
@@ -37,4 +43,8 @@ CREATE TABLE IF NOT EXISTS cache.dependencies
                                     REFERENCES cache.cached (query_id) MATCH SIMPLE
                                     ON UPDATE NO ACTION
                                     ON DELETE CASCADE
-                            );
\ No newline at end of file
+                            );
+
+CREATE TABLE cache.cache_config (key text, value text);
```

Comment:
Shouldn't the `value` be numeric?

Reply:
Well there are already two types in there - half_life is a float, cache_size is an integer number of bytes. But yes, also allows for other types if more settings should be added.

Reply:
Oh sorry, I didn't realise that `half_life` is a float because in the docs (and when it is initialised) this is done with integer values.